cypress/e2e/cloud/helpBar.test.ts

```diff
@@ -3,10 +3,7 @@ describe('Help bar support for free account users', () => {
     cy.flush().then(() =>
       cy.signin().then(() => {
         cy.get('@org').then(() => {
-          cy.setFeatureFlags({
-            helpBarSfdcIntegration: true,
-            uiUnificationFlag: true,
```

Comment:
FYI - I could be wrong since not a flag I deployed - but I think the `uiUnificationFlag` still needs to remain on in this test.